src/components/periodic-report/periodic-report.repository.ts

```diff
@@ -30,15 +30,15 @@ import {
 @Injectable()
 export class PeriodicReportRepository extends DtoRepository(IPeriodicReport) {
   async create(input: CreatePeriodicReport) {
-    const reportFileId = await generateId();
+    const reportDirectoryId = await generateId();
 
     const Report = resolveReportType(input);
     const initialProps = {
       type: input.type,
       start: input.start,
       end: input.end,
       receivedDate: null,
-      reportFile: reportFileId,
+      reportDirectory: reportDirectoryId,
```

Comment:
I know this is the way that we do defined files, but it doesn't really make much sense. We end up having both:
```cql
(:BaseNode)-[:reportFile]->(:Property { id: 'asdf' })
(:BaseNode)-[:reportFileNode]->(:File { id: 'asdf' })
```
I think there was a period of time in dev where the file wasn't created until the first version was uploaded.

Since we aren't bound by the _defined file_ abstractions here, let's just make this a normal relationship.
The directory can be created first and then attached with `createRelationships`.

This does mean adding an explicit match in the hydrate query.